internal/testing/e2e/licensemanager/license.go

```diff
@@ -0,0 +1,39 @@
+//NOTE - this code is copied directly from this file https://github.com/hashicorp/consul-enterprise/blob/6099c553634223a9a7ddf2263d90eeaef98c50e0/agent/licensemanager/loader.go#L24-L36
+// the package isn't being used directly as part of a conscious choice to limit using consul-enterprise as a direct dependency
+
```

Comment:
I'm actually fine dropping this comment, it's not going to make sense to any non-internal developers hacking on this and things are simple enough that I don't mind drift in this code.